tuxemon/states/pc_kennel/__init__.py

```diff
@@ -192,16 +192,16 @@ def description(mon: Monster) -> None:
             label = T.translate(monster.name).upper()
             iid = monster.instance_id.hex
             new_image = pygame_menu.BaseImage(
-                transform_resource_filename(monster.menu_sprite_1),
+                transform_resource_filename(monster.front_battle_sprite),
                 drawing_position=POSITION_CENTER,
             )
-            new_image.scale(prepare.SCALE, prepare.SCALE)
+            new_image.scale2x()
```

Comment:
why changed to scale 2x?

Reply:
@bitcraft because the icons needs to be smaller, there are 4 columns

Reply:
we should be using ratios based on scale, because if you change the screen resolution, the SCALE value changes.  therefore, if we hardcode the scale value, the game would have different sized sprites at different resolutions

Reply:
it might look correct at the resolution you are using, but if it is run at some different resolution, the icons will be the wrong size

Reply:
ok

Reply:
if you want them to be a specific size, you can do something such as `icon_size * (SCALE * .5)`, which would keep the icons the correct size as the screen resolution changes.

Reply:
you are right, I'm going to put *.5

without `*.5`
![Screenshot_2023-11-04_18-00-00](https://github.com/Tuxemon/Tuxemon/assets/64643719/0a406f19-5f43-4808-a142-d2472627ec86)

with` *.5`
![Screenshot_2023-11-04_18-02-46](https://github.com/Tuxemon/Tuxemon/assets/64643719/a2c80a7b-b02a-42e2-9120-77274122ec94)